scripts/build_posix.sh

```diff
@@ -51,6 +51,7 @@ command_exists()
 
 fpm_build()
 {
+	echo "PKG = \"$1\"" > $(pwd)/dvc/utils/build.py
```

Comment:
This is not going to work, as dvc is already built in `build_dvc`. Please see my comment https://github.com/iterative/dvc/pull/2773#discussion_r344504562